### PR TITLE
Misc. fixes to CC RN, sidebars, custom RBAC

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

**Comment by ctauchen** [comment]
@girishshankaran Can you take a look?

File: calico-cloud/release-notes/index.mdx [hunk 1]
```diff
@@ -14,21 +14,33 @@ Calico Cloud includes new and improved Dashboards that enable operators to defin
...
 
-## Namespace-based policy recommendations
+For more information, see [Manager UI tutorial](../tutorials/calico-cloud-features/tour.mdx#dashboard).
+
+### Namespace-based policy recommendations
 
 Calico Cloud has improved its policy recommendation engine to add namespace-based recommendations.
 This enables operators to easily implement microsegmentation for namespaces.
 
+For more information, see [Create policy recommendation](.../network-policy/generate-policy-recommendation.mdx).
```
Comment (girishshankaran) [inline calico-cloud/release-notes/index.mdx]: This link is not working in the preview output.

**Review by girishshankaran — DISMISSED** [review]
LGTM except the bad link.

File: calico-cloud/release-notes/index.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-cloud/release-notes/index.mdx]: Fixed, thanks!

**Pull request merged**